src/components/ui/tag.tsx

```diff
@@ -0,0 +1,88 @@
+import { cva, VariantProps } from "class-variance-authority"
+import { Slot } from "@radix-ui/react-slot"
+
+import { cn } from "@/lib/utils/cn"
+
+const tagVariants = cva(
+  "inline-flex items-center rounded-full border px-2 py-0.5 min-h-8 text-xs uppercase transition-colors focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2",
+  {
+    variants: {
+      status: {
+        normal: "bg-background-highlight text-body-medium border-body-medium",
+        tag: "bg-primary-low-contrast text-primary border-primary",
+        success: "bg-success-light text-success border-success-border",
+        error: "bg-error-light text-error border-error-border",
+        warning: "bg-warning-light text-warning-dark border-warning-border",
+      },
+      variant: {
+        subtle: "border-transparent",
+        solid: "border-transparent text-body-inverse",
+        outline: "bg-transparent",
+      },
```

Comment:
Any reason we aren't adding the `highContrast` variant shown in DS?

<img width="726" alt="image" src="https://github.com/user-attachments/assets/d5e4d59e-4bbe-4ced-938b-090840b68e15">

Reply:
Ahh I completely miss this case. ill add it.